### PR TITLE
Manual Merge of PR 104 to a new branch, then changed method of burn

### PR DESCRIPTION
**Pull request by jrwashburn**
Merged MF fork from #104, then applied changes to the method of burning - instead of adding new methods in the transaction sender to construct a burn transaction, added the burn transaction to the list of payouts - treating it just like any other transaction to be sent.

I think this is a simpler approach and allows the burn to be seen in the payout details and transaction list explicitly. 

Tested with a real pool and different shareclasses; will run another set of tests on historical pool data to confirm before merge.

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -5,8 +5,8 @@ BLOCK_DATA_SOURCE=MINAEXPLORER
 
 # Pool Configuration
 COMMISSION_RATE=.05
-MF_COMMISSION_RATE=0.05
-O1_COMMISSION_RATE=0.05
+MF_COMMISSION_RATE=0.08
+O1_COMMISSION_RATE=0.08
```
Comment (snaitmouloud-mf) [inline sample.env:9]: O1 COMMISSION RATE should be: 0.05
https://blog.o1labs.org/o-1-labs-delegation-policy-786bf96f9fdd

**Review by snaitmouloud-mf — COMMENTED** [review]
O1 COMMISSION RATE should be: 0.05
https://blog.o1labs.org/o-1-labs-delegation-policy-786bf96f9fdd

**Comment by snaitmouloud-mf** [comment]
Except the small minor change above, All good for me to merge.

File: sample.env (hunk 1 above)
Comment (jrwashburn) [reply sample.env]: thanks - fixed

Comment (jrwashburn) [reply sample.env]: updated

**Review by jrwashburn — COMMENTED** [review]
Updated sample.env o1 rate; ready to merge.

**Comment by jrwashburn** [comment]
I'm also going to get the tests passing - not sure I'll mock up enough data to test the burn, but will at least get them passing again. I need to write something that generates the necessary test data at some point.

**Comment by jrwashburn** [comment]
> I'm also going to get the tests passing - not sure I'll mock up enough data to test the burn, but will at least get them passing again. I need to write something that generates the necessary test data at some point.

Or at least some of them. I think this is ready to merge; will revisit tests when I have more time.

**Pull request merged**